srcpkgs/genymotion/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'genymotion'
+pkgname=genymotion
+version=3.0.4
+revision=1
+archs="x86_64"
+_full_name="${pkgname}-${version}-linux_x64.bin"
+create_wrksrc=yes
+build_style=fetch
+hostmakedepends="tar which"
+depends="libpng net-tools protobuf qca-qt5 qt5-script qt5-webkit virtualbox-ose"
+short_desc="Complete set of tools that provides a virtual environment for Android"
+maintainer="linarcx <linarcx@riseup.net>"
+license="custom:Proprietary"
```

Comment:
Need to vlicense the file

Reply:
there's not license file included in downloaded distfile.

Reply:
It is not redistributable, https://www.genymotion.com/legal/terms-and-conditions/ . Chances to include this are rather low, even as `restricted`.